planning/behavior_path_planner/src/scene_module/pull_out/pull_out_module.cpp

```diff
@@ -29,13 +29,38 @@
 #include <utility>
 #include <vector>
 
+using motion_utils::calcLongitudinalOffsetPose;
+using tier4_autoware_utils::calcOffsetPose;
 namespace behavior_path_planner
 {
 PullOutModule::PullOutModule(
   const std::string & name, rclcpp::Node & node, const PullOutParameters & parameters)
-: SceneModuleInterface{name, node}, parameters_{parameters}
+: SceneModuleInterface{name, node},
+  parameters_{parameters},
+  clock_{node.get_clock()},
```

Comment:
interface already has clock_.

Reply:
fixed in fixed in https://github.com/autowarefoundation/autoware.universe/pull/1438/commits/74f1feb7410fe50e81887d9563cbb9c24fe95751